main.c

```diff
@@ -69,6 +69,8 @@ bool extprefix;
 
 bool resized = false;
 
+XEvent ev;
```

Comment:
If it's not being accessed outside of `main.c` then it should be `static.` Or I can do it in #146 if we want to accept that PR.

Reply:
I can be accessed by commands.c (and we plan to for the thumbnail binding which was part of the point)

Reply:
> we plan to for the thumbnail binding

On second thought, doesnt it then make more sense to do it on the thumb bindings PR then?

Reply:
Perhaps. It also affects #145 and technically has some benefit in extensibility on its own. This PR doesn't really have a clear focus so I don't have a strong opinion. I'd just leave it it in #145 if it got reverted here

Reply:
Let's do it on the relevant PR then, either #145 if we accept that or thumb binding.

> This PR doesn't really have a clear focus

We can focus on `process_bindings`. Feel like wildcard can be it's own PR, though no strong opinion.